source/images/right-chevron.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd want to hear from @cadecairos or @Pomax on this, should these live here, or maybe under images/glyps? Or are there any icon dependencies in the project that might already have chevron arrows for the links?

Reply:
looks like there's already a lot of disorganization in `source/images/`. However, I do see a `down-chevron.svg` in `glyphs` so we should probably move this there.